projects/js-packages/publicize-components/src/social-store/hydrate-stores.ts

```diff
@@ -70,16 +70,6 @@ export async function hydrateStores() {
 				label: __( 'Publicize shares data', 'jetpack-publicize-components' ),
 			},
 		] );
-
-		// @ts-expect-error Only 3 arguments are required, rest are optional but types expect 7
-		await receiveEntityRecords(
-			'wpcom/v2',
-			'publicize/shares-data',
-			getSocialScriptData()?.shares_data,
-			true
-		);
```

Comment:
We might still want to do this I think, so that, to begin with, the cards will show the cached values. That might need some testing though!

Reply:
They do show the values without this as per my testing